docs/content/getting-started/production-environment.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: Production Best Practices
```

Comment:
Do you want to rename this file to match this title as well?

Reply:
Done